acceptance/tests/facts/non_root_users_without_errors.rb

```diff
@@ -0,0 +1,34 @@
+test_name "C59196: running facter as a non-root user should not produce permission errors" do
+  tag 'risk:high'
+
+  # confine :except, :platform => 'aix' # bug FACT-1586
+
+  confine :except, :platform => 'windows' # this test is for unix systems
```

Comment:
Why does this only apply to unix systems? Don't we support running `facter` as a non-privileged user on Windows?